hack/test-smoke.sh

```diff
@@ -54,6 +54,9 @@ EOF
 	INFO "Waiting for 3 replicas to be ready"
 	kubectl rollout status --timeout=5m statefulset
 
+	INFO "Ensure that kubectl exec works"
+	for f in {0..2}; do kubectl exec "dnstest-${f}" -- hostname; done
```

Comment:
@vsoch I can't repro the issue

Reply:
I'll see if I can generate a test case - I was running an experiment today and didn't have the time!